Flow.Launcher/SettingPages/ViewModels/SettingsPaneThemeViewModel.cs

```diff
@@ -94,30 +96,9 @@ public double ResultSubItemFontSize
     public List<string> Themes =>
         ThemeManager.Instance.LoadAvailableThemes().Select(Path.GetFileNameWithoutExtension).ToList();
 
+    public class ColorSchemeData : DropdownDataGeneric<ColorSchemes> { }
 
-    public class ColorScheme
-    {
-        public string Display { get; set; }
-        public ColorSchemes Value { get; set; }
-    }
-
-    public List<ColorScheme> ColorSchemes
-    {
-        get
-        {
-            List<ColorScheme> modes = new List<ColorScheme>();
-            var enums = (ColorSchemes[])Enum.GetValues(typeof(ColorSchemes));
-            foreach (var e in enums)
-            {
-                var key = $"ColorScheme{e}";
-                var display = InternationalizationManager.Instance.GetTranslation(key);
-                var m = new ColorScheme { Display = display, Value = e, };
-                modes.Add(m);
-            }
-
-            return modes;
-        }
-    }
+    public List<ColorSchemeData> ColorSchemes { get; } = DropdownDataGeneric<ColorSchemes>.GetValues<ColorSchemeData>("ColorScheme");
```

Comment:
This may have the issue that it may not reflect the language change in flight? Are the previous version able to handle that?

Reply:
The old version also didn't handle language change. In this case it doesn't matter though, because the language change can only happen in the General tab. When the user goes back to the Appearance tab after that, it will re-render completely, including those checkboxes.